Model/Quote/Checkout.php

```diff
@@ -97,6 +97,20 @@ public function getCheckoutSettings()
         ];
     }
 
+    /**
+     * @return float|int
+     */
+    private function getDeliveryDelayDay()
```

Comment:
Waarom heet dit zo? Het haalt iets op wat we dropoff delay noemen dus iets als `getDropOffDelayByProducts` zou misschien logischer zijn

Reply:
En kan deze functie niet beter in `$this->helper`?

Reply:
done